ios/FluentUI/List/FluentListSectionHeader.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+/// This component is a work in progress. Expect changes to be made to it on a somewhat regular basis.
+///
+/// It is intended to be used in conjunction with `FluentUI.FluentListSection` and `FluentUI.ListItem`
+public struct FluentListSectionHeader<Title: StringProtocol, TrailingContent: View>: View {
+
+    // MARK: Initializer
+
+    /// Creates a `FluentListSectionHeader`
+    /// - Parameters:
+    ///   - title: title of the section to be shown in the header.
+    ///   - trailingContent: content that appears on the trailing edge of the view.
+    public init(title: Title,
+                @ViewBuilder trailingContent: @escaping () -> TrailingContent) {
+        self.title = title
+        self.trailingContent = trailingContent
+    }
+
+    public var body: some View {
+        @ViewBuilder var titleView: some View {
+            Text(title)
+                .textCase(nil)
```

Comment:
`nil` is the default value for `.textCase(_:)`, right?

Reply:
yeah its a bit odd, but if you don't provide it the default casing for the header is all caps, so yes it is necessary. Seems like uppercase textCase is being applied automatically even though the API doesn't specify that

Reply:
done!